<!DOCTYPE UI>
<ui version="4.0" >
 <class>chapterEdit</class>
 <widget class="QWidget" name="chapterEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Chapter properties</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="7" row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="image" >
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QSlider" name="slider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="lPosition" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>--:--:--</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>video</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3" >
    <widget class="KUrlRequester" name="urFile" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="filter" >
      <string>*.avi
*.mpeg
*.mpg
*.*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>start</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QTimeEdit" name="twstart" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat" >
      <string>HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="KPushButton" name="bStart" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>set Current Frame</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="2" >
    <widget class="QCheckBox" name="ckBreakStart" >
     <property name="text" >
      <string>break continuity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>end</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QTimeEdit" name="twend" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat" >
      <string>HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="KPushButton" name="bEnd" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>set Current Frame</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2" >
    <widget class="QCheckBox" name="ckBreakEnd" >
     <property name="text" >
      <string>break continuity</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>21</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" >
    <widget class="QPushButton" name="bTitleButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>set Title button</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>urFile</sender>
   <signal>urlSelected(const KUrl&amp;)</signal>
   <receiver>chapterEdit</receiver>
   <slot>urlSelected(const KUrl&amp;)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>sliderReleased()</signal>
   <receiver>chapterEdit</receiver>
   <slot>sliderReleased()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chapterEdit</receiver>
   <slot>sliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bStart</sender>
   <signal>clicked()</signal>
   <receiver>chapterEdit</receiver>
   <slot>bStartClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bEnd</sender>
   <signal>clicked()</signal>
   <receiver>chapterEdit</receiver>
   <slot>bEndClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twstart</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>chapterEdit</receiver>
   <slot>startChanged(QTime)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twend</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>chapterEdit</receiver>
   <slot>endChanged(QTime)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckBreakEnd</sender>
   <signal>clicked()</signal>
   <receiver>chapterEdit</receiver>
   <slot>ckBreakEndClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckBreakStart</sender>
   <signal>clicked()</signal>
   <receiver>chapterEdit</receiver>
   <slot>ckBreakStartClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bTitleButton</sender>
   <signal>clicked()</signal>
   <receiver>chapterEdit</receiver>
   <slot>bTitleButtonClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
